apps/server/src/controllers/main.controller.js

```diff
@@ -1,8 +1,21 @@
 const mainService = require('../services/main.service');
 
 const signIn = async (req, res) => {
-    // const plants = await mainService.signIn();
-    res.send('sign-in page is under construction');
+    const { phoneNumber } = req.body;
+    if (!phoneNumber) {
```

Comment:
S: validate the given value. you can copy from client sign-in.validator.ts file

Reply:
In progress

Reply:
Validation still in progress.